src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -162,6 +162,23 @@ Namespace Microsoft.CodeAnalysis.Operations
                 isImplicit:=isImplicit OrElse argument Is Nothing)
         End Function
 
+        Private Function CreateReceiverOperation(node As BoundNode, symbol As ISymbol) As IOperation
```

Comment:
Consider returning `Lazy(Of IOperation)` instead and use singleton if no receiver, to match C#.

Reply:
Done.

---
In reply to: [149578417](https://github.com/dotnet/roslyn/pull/22972#discussion_r149578417) [](ancestors = 149578417)